scipy/special/tests/test_zeta.py

```diff
@@ -52,86 +52,200 @@ def test_riemann_zeta_avoid_overflow():
-    #     y = y_sign * rng.uniform(2, 8)
-    #     z = x + y*1j
-    #     reference = complex(mp.zeta(z))
-    #     cases.append((z, reference))
-
-    # # Moderately large imaginary part in each quadrant outside of critical strip
-    # for x_sign, y_sign in [1, 1], [1, -1], [-1, 1], [-1, -1]:
-    #     x = x_sign * rng.uniform(2, 8)
-    #     y = y_sign * rng.uniform(50, 80)
-    #     z = x + y*1j
-    #     reference = complex(mp.zeta(z))
-    #     cases.append((z, reference))
-
-    # # points in critical strip
-    # x = rng.uniform(-1.0, 1.0, size=5)
-    # y = np.exp(rng.uniform(0, 5, size=5))
-    # z = x + y*1j
-    # for t in z:
-    #     reference = complex(mp.zeta(t))
-    #     cases.append((complex(t), reference))
-    # z = x - y*1j
-    # for t in z:
-    #     reference = complex(mp.zeta(t))
-    #     cases.append((complex(t), reference))
+    "z, desired, rtol",
     [
-        # small values in each quadrant outside critical strip
+        ## Test cases taken from mpmath with the script:
+
+        # import numpy as np
+        # import scipy.stats as stats
+
+        # from mpmath import mp
+
+        # # seed = np.random.SeedSequence().entropy
+        # seed = 154689806791763421822480125722191067828
+        # rng = np.random.default_rng(seed)
+        # default_rtol = 1e-13
+
+        # # A small point in each quadrant outside of the critical strip
+        # cases = []
+        # for x_sign, y_sign in [1, 1], [1, -1], [-1, 1], [-1, -1]:
+        #     x = x_sign * rng.uniform(2, 8)
+        #     y = y_sign * rng.uniform(2, 8)
+        #     z = x + y*1j
+        #     reference = complex(mp.zeta(z))
+        #     cases.append((z, reference, default_rtol))
+
+        # # Moderately large imaginary part in each quadrant outside of critical strip
+        # for x_sign, y_sign in [1, 1], [1, -1], [-1, 1], [-1, -1]:
+        #     x = x_sign * rng.uniform(2, 8)
+        #     y = y_sign * rng.uniform(50, 80)
+        #     z = x + y*1j
+        #     reference = complex(mp.zeta(z))
+        #     cases.append((z, reference, default_rtol))
+
+        # # points in critical strip
+        # x = rng.uniform(0.0, 1.0, size=5)
+        # y = np.exp(rng.uniform(0, 5, size=5))
+        # z = x + y*1j
+        # for t in z:
+        #     reference = complex(mp.zeta(t))
+        #     cases.append((complex(t), reference, default_rtol))
+        # z = x - y*1j
+        # for t in z:
+        #     reference = complex(mp.zeta(t))
+        #     cases.append((complex(t), reference, default_rtol))
+
+        # # Near small trivial zeros
+        # x = np.array([-2, -4, -6, -8])
+        # y = np.array([1e-15, -1e-15])
+        # x, y = np.meshgrid(x, y)
+        # x, y = x.ravel(), y.ravel()
+        # z = x + y*1j
+        # for t in z:
+        #     reference = complex(mp.zeta(t))
+        #     cases.append((complex(t), reference, 1e-7))
+
+        ## Some other points near real axis
```

Comment:
```suggestion
        # # Some other points near real axis
```
I only happened to notice this because of the way I was removing the `#` (on Colab)